static/katex.less

```diff
@@ -10,6 +10,7 @@
 
     > .katex {
         display: inline-block;
+        text-align: initial;
```

Comment:
Yeah probably. Or `start`, falling back to `left`?

https://davidwalsh.name/text-align-start

Not sure KaTeX works well with RTL yet though… :)